configs/conda.config

```diff
@@ -11,13 +11,7 @@ process {
     withLabel: sortmerna    { conda = "$baseDir/envs/sortmerna.yaml" } 
     withLabel: trinity      { conda = "$baseDir/envs/trinity.yaml" } 
     withLabel: stringtie    { conda = "$baseDir/envs/stringtie.yaml" } 
-
-    // unfortunately no working conda envs? thus use docker
-    //withLabel: busco { conda = "$baseDir/envs/busco.yaml" } 
-    //withLabel: dammit { conda = "$baseDir/envs/dammit.yaml" }
-
-    withLabel: busco { container = 'nanozoo/busco:3.0.2--40d1506' } 
-    withLabel: dammit { container = 'nanozoo/dammit:1.2--b47259e' } 
-
-    withLabel: basic_tools {conda = "$baseDir/envs/basic_tools.yaml" }   
-}
\ No newline at end of file
+    withLabel: busco        { conda = "$baseDir/envs/busco.yaml" } 
+    withLabel: basic_tools  { conda = "$baseDir/envs/basic_tools.yaml" }
+    withLabel: dammit       { conda = "$baseDir/envs/dammit.yaml" }
```

Comment:
does the dammit conda env actually work? I remember issues back then... but would need to test it w/ conda actually @fischer-hub

Reply:
ah yeah,I see that you put the full yml dependency file to get it working. Also testing this right now but looks good so far!

Reply:
Yes I think for the dammit and deseq2 environments I had to put the full dependencies to get it to run at all. But I got those from inside the docker containers, so should be pretty much the same installation.

Reply:
yep, both worked for me using conda! So that's fine